.github/workflows/ci.yaml

```diff
@@ -68,8 +68,26 @@ jobs:
       - uses: Swatinem/rust-cache@v1
       # Needed for agent
       - run: cargo fmt --all -- --check
+      - run: rustup target add --toolchain nightly aarch64-apple-darwin
```

Comment:
Does this have an affect? `--all-targets` is not needed for that?

Reply:
oops, my bad.